projects/ngx-turnstile-demo/src/app/examples/template-driven-form/template-driven-form-example.component.ts

```diff
@@ -9,6 +9,8 @@ import { NgxTurnstileModule, NgxTurnstileFormsModule } from 'ngx-turnstile';
     ><ngx-turnstile
       [siteKey]="siteKey"
       theme="light"
+      retry="never"
+      size="normal"
```

Comment:
Can you remove the new inputs here.

```suggestion
```

And then add a new route to our demo app which shows the different sizes.

In this folder:
https://github.com/verto-health/ngx-turnstile/tree/main/projects/ngx-turnstile-demo/src/app/examples